pkg/binary/envoy/config.go

```diff
@@ -97,7 +97,7 @@ func (r *Runtime) SaveConfig(name, config string) (string, error) {
 		return "", fmt.Errorf("Unable to create directory %q: %v", configDir, err)
 	}
 	filename := name + ".yaml"
-	err := ioutil.WriteFile(filepath.Join(configDir, filename), []byte(config), 0644)
+	err := ioutil.WriteFile(filepath.Join(configDir, filename), []byte(config), 0600)
```

Comment:
new version of the sec linter is more strict. I went with the advice instead of ignoring it.